solana/crates/kolme-solana-bridge/src/lib.rs

```diff
@@ -50,17 +50,20 @@ const GIT_REV: &str = "N/A";
 // PDAs
 const STATE_DERIVATION: PdaDerivation = PdaDerivation::new(&[STATE_SEED]);
 const GIT_REV_DERIVATION: PdaDerivation = PdaDerivation::new(&[GIT_REV_SEED]);
+const UPGRADE_AUTHORITY_DERIVATION: PdaDerivation = PdaDerivation::new(&[UPGRADE_AUTHORITY_SEED]);
 
 type StatePda = PdaData<State, 0>;
 type TokenHolderPda = PdaData<TokenHolder, 1>;
 type GitRevPda = PdaData<GitRev, 2>;
+type UpgradeAuthorityPda = PdaData<(), 3>;
 
 const SHA256_DIGEST_SIZE: usize = 32;
 
 #[repr(u32)]
 pub enum InitIxError {
-    NoApproversProvided = 0,
```

Comment:
Will changing the representation of these values present any backwards-incompatibility issues with existing deployed contracts?

Reply:
No, those only show up in transaction results. In practice, since transactions are first simulated, they won't even end up being recorded anywhere.